### ISSUE TITLE
Paper builder

### ISSUE THREAD

**Issue by CJ-Wright** [regro/regolith#132]
It may be nice to have regolith build papers.
The idea here would be to track the relevant information (authors, title, etc) in the databases and have jinja2 formatting in the paper. The builder would then fill in the relevant information (including author emails and institutions, grant/experiment acknowledgements, etc.), and build the PDF of the paper. It may be possible
Regolith would then "release" the paper via rever and then send around an email to all the authors with the PDF attached for final review.
If there are APIs for submitting papers to journals/arxive we could send up the relevant info.

tl;dr turn key paper building/author management/submission.
